auth/user_mgt.go

```diff
@@ -416,6 +424,13 @@ func validatePhone(val interface{}) error {
 	return nil
 }
 
+func validateValidSince(val interface{}) error {
+	if _, ok := val.(int64); !ok {
+		return fmt.Errorf("tokens valid after time must be an integer")
+	}
```

Comment:
Also check for non-negative.

Reply:
if 1971 is an ok date, why not 1969 ?
(should we add a check that the epoch seconds are e.g. > 1e9 (~Y2K))

Reply:
I'd just do a simple sign check to begin with. We can implement more restrictive checks later if it becomes necessary.